students/timm67/session08/circle.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+"""
+Tim Meese
+Au2018-Py210B
+Circle Lab
+"""
+
+from math import pi
+
+
+class Circle():
+    _radius = float(0.0)
+    _diameter = float(0.0)
+    _area = float(0.0)
+    _circumference = float(0.0)
```

Comment:
Not sure why you defined these class attributes, when they better/should be instance attributes that should be defined in `__init__()` (which you did). Probably these lines (13-16) can/should be removed.

Reply:
Addressed.